.github/workflows/misspell.yml

```diff
@@ -0,0 +1,18 @@
+name: reviewdog
+on: [pull_request]
+jobs:
+  misspell:
+    name: runner / misspell
+    runs-on: ubuntu-latest
+    steps:
+      - name: Check out code.
+        uses: actions/checkout@v2
+      - name: misspell
+        uses: reviewdog/action-misspell@v1
```

Comment:
this seems somewhat unknown repo (https://github.com/reviewdog/action-misspell) and it seems to have github_token available to it.

- is there more "common" action for this
- can we lock action to digest so that it doesnt "change" under us without review
- can we eliminate/scope what the token can do? (what can it do today?)

Reply:
It looks like we can get away without the token and it will log the errors instead of posting a review